spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/RabbitConnectionFactoryBean.java

```diff
@@ -629,4 +626,36 @@ protected SSLContext createSSLContext() throws NoSuchAlgorithmException {
 		return SSLContext.getInstance(this.sslAlgorithm);
 	}
 
+
+	/**
+	 * In case the client doesn't provide the sslProperties location or the truststore is null delegate to Java's default
+	 * trust mechanism
+	 * @throws NoSuchAlgorithmException
+	 * @throws KeyManagementException
+	 * @throws KeyStoreException
+     */
+	public void useDefaultTrustStoreMechanism() throws NoSuchAlgorithmException, KeyManagementException,
+			KeyStoreException {
+		SSLContext c = SSLContext.getInstance(this.sslAlgorithm);
+		c.init(null, new TrustManager[] {  getDefaultTrustManager(TrustManagerFactory.getDefaultAlgorithm(), null) }, null);
+		this.connectionFactory.useSslProtocol(c);
+	}
+
+
+	public static X509TrustManager getDefaultTrustManager(String algorithm, KeyStore keystore) throws NoSuchAlgorithmException, KeyStoreException {
```

Comment:
Even if it is fine to have it as a `static` method, I'm not sure that it is good to `public` it anyway.
The same `public` concern for the `useDefaultTrustStoreMechanism()` method above.